### DISCUSSION TITLE
Lazyterm highlight

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#4665] [General]
How do I configure the highlight for the terminal window by executing `<c-/>`? E.g. I want to add a color to the border.

**Comment by dpetka2001** [comment]
It's the highlight group `FloatBorder`.

**Reply by idelice** [reply]
I also forgot to ask if it was possible to add borders to it?

**Reply by dpetka2001** [reply]
There are 2 places you can do that. 

Either in your `~/.config/nvim/lua/config/lazy.lua` add 
```lua
ui = {
  border = "rounded"    -- for more options of what value the `border` accepts look at `:h nvim_open_win()`
},
```
This will change all the windows that `lazy.nvim` creates to have border according to your setting (this does not affect Mason window because it creates its own window).

Or if you want to change it specifically only for the terminal, then you have to change the keymap in your personal configuration
```lua
vim.keymap.set("n", "<C-/>", function()
  LazyVim.terminal(nil, { border = "rounded" })
end, { desc = "Open terminal" })
```
Note that changing the `FloatBorder` highlight group it will change the highlight color for all borders in every window I believe.

**Reply by idelice** [reply]
It did the job, but holy crap that border is huge 😆 

<img width="2166" alt="Screenshot 2024-10-30 at 10 39 33" src="https://github.com/user-attachments/assets/d5eb67f2-3141-44b1-ab76-3348b078b5ef">

**Reply by dpetka2001** [reply]
You probably gave a `bg` color instead of a `fg` color. `bg` will fill in the whole cell of the terminal whereas `fg` will only highlight the characters that the border consists of.

This is how I do it on tokyonight colorscheme that I use (I left the default `bg` from tokyonight unchanged and only changed the `fg`)
```lua
  {
    "folke/tokyonight.nvim",
    style = "moon",
    opts = {
      on_highlights = function(hl, _)
        hl.GitSignsAdd = { fg = "#006C00" }
        hl.GitSignsChange = { fg = "#FF8000" }
        hl.FloatBorder = { fg = "red", bg = "#1e2030" }
      end,
    },
  },
```
and put `ui = { border = "rounded }` in `lua/config/lazy.lua` and it looks like this

![2024-10-30_17-34](https://github.com/user-attachments/assets/0f86332f-c3f7-404e-8f51-0efff40d1d9a)
